rocket-chat-android-widgets/src/main/res/layout/message_avatar.xml

```diff
@@ -4,4 +4,5 @@
         android:id="@+id/drawee_avatar"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        fresco:actualImageScaleType="fitCenter" />
\ No newline at end of file
+        fresco:actualImageScaleType="fitCenter"
+        fresco:roundedCornerRadius="5dp" />
```

Comment:
Oh, sounds good! I was thinking it would not use because of the name `message_avatar`. Thank you! 👍